### PR TITLE
Amounts inputs redesign

### PR DESCRIPTION
**Pull request by herou**
@TalgatNurdinov Please check this one so I can go further with the other layouts which need the same redesign.

### CODE REVIEW

**Comment by abdrasulov** [comment]
Screenshot_1608098540

Please check the paddings on top of Amount and left of ZRX

**Comment by herou** [comment]
@abdrasulov  I checked on all devices before pushing and I did not have this strange behavior. Which device are you using? I made another commit, please check it.

**Comment by abdrasulov** [comment]
@herou Now it looks good. I checked it on Samsung A6.

**Comment by herou** [comment]
@abdrasulov I will make the same changes on the other layout.

**Comment by herou** [comment]
@abdrasulov U have it.

**Review by abdrasulov — APPROVED** [review]

File: app/src/main/java/io/horizontalsystems/bankwallet/core/providers/AppConfigProvider.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ class AppConfigProvider : IAppConfigProvider, ILanguageConfigProvider, IAppConfi
     override val guidesUrl = App.instance.getString(R.string.guidesUrl)
     override val faqUrl = App.instance.getString(R.string.faqUrl)
     override val fiatDecimal: Int = 2
-    override val maxDecimal: Int = 8
+    override val maxDecimal: Int = 9
```
Comment (abdrasulov) [inline app/src/main/java/io/horizontalsystems/bankwallet/core/providers/AppConfigProvider.kt:35]: Hi @herou. We will work on the logic of decimals by another issue. At the moment, please set `maxDecimal` to 8

**Review by omurovch — APPROVED** [review]

**Review by tmedetbekov — APPROVED** [review]

**Pull request merged**